src/main/java/nl/tudelft/ewi/devhub/server/backend/CoursesBackend.java

```diff
@@ -33,7 +35,10 @@
 public class CoursesBackend {
 
     @Inject
-    private CourseEditions courses;
+    private CourseEditions courseEditions;
```

Comment:
Nit picky, but how do you feel about changing the field injection to constructor injection?
But if we want to remove that let's do so in a different pr.